File: VB/RibbonDesigner/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Windows.Forms
Imports DevExpress.XtraReports.Design
Imports DevExpress.XtraReports.UserDesigner
' ...

Namespace RibbonDesigner
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			Dim rep As New XtraReport1()
			Dim form As New XRDesignRibbonFormEx()
			form.DesignPanel.OpenReport(rep)
			form.ShowDialog()
			Close()
		End Sub
	End Class
End Namespace

File: CS/RibbonDesigner/Form1.cs
using System;
using System.Windows.Forms;
using DevExpress.XtraReports.Design;
using DevExpress.XtraReports.UserDesigner;
// ...

namespace RibbonDesigner {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e) {
            XtraReport1 rep = new XtraReport1();
            XRDesignRibbonFormEx form = new XRDesignRibbonFormEx();
            form.DesignPanel.OpenReport(rep);
            form.ShowDialog();
            Close();
        }
    }
}